core_contracts/stakedLp/interfaces.py

```diff
@@ -0,0 +1,60 @@
+from iconservice import *
+
+
+class Status:
+    STAKED = 1
+
+
+class SupplyDetails(TypedDict):
+    decimals: int
+    principalUserBalance: int
+    principalTotalSupply: int
+
+
+class TotalStaked(TypedDict):
+    decimals: int
+    totalStaked: int
+
+
+class AddressDetails(TypedDict):
+    name: str
+    address: Address
+
+class RewardsDataEntry(TypedDict):
+    _user: Address
+    _balance: int
+
+class AssetConfig(TypedDict):
+    _id: int
+    asset: Address
+    distPercentage: int
```

Comment:
How is AssetConfig used? What is an Asset here? Is distPercentage the same as the distPercentage in rewards? If so, how do they stay in sync?